playbooks/infra/deploy-vm-bastion-libvirt.yml

```diff
@@ -321,3 +321,14 @@
           - java
         state: present
       become: true
+
+    - name: Ensure CA certifications are updated
```

Comment:
I think these tasks should be optional. Not sure if we will always have access to this certificate

Reply:
The bastions and BMs always needs this certificate as they are the executors and the code in hosted on gitlab (requires the certificate).

Reply:
Just trying to be prepared in case we run this in different lab environments